include/fs/fs.h

```diff
@@ -307,6 +311,10 @@ int fs_unlink(const char *path);
  * @param to The destination path
  *
  * @retval 0 on success;
+ * @retval -EINVAL when a bad file name is given;
+ * @retval -EINVAL when rename would cause move between mount points;
```

Comment:
-EINVAL is described twice

Reply:
That is intentional because there are two possible causes, from VFS level, to return EINVAL here; I have tried to do same thing as the `man open` do, where different causes for EINVAL are listed separately, but I may merge these two lines together if you wish.

Reply:
Correction: It seems I have to do the merge of lines as documentation generating process reports this as problem.